src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -225,4 +231,39 @@ public String toString() {
                 + this.hasTableOfContents + ", hasCoversheets=" + this.hasCoversheets + ", hasFolderCoversheets="
                 + this.hasFolderCoversheets + ")";
     }
+
+    @Transient
+    public Integer getSubtiles(SortableBundleItem container, Map<BundleDocument, File> documents) {
+        return container
+                .getSortedItems().flatMap(SortableBundleItem::getSortedDocuments)
+                .map(i -> {
+                    try {
+                        return load(documents.get(i)).getDocumentCatalog().getDocumentOutline();
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    return null;
+                })
+               .filter(o->o!=null)
+               .filter(o->o.getFirstChild()!=null)
+               .mapToInt(o -> o!=null? getItemsFromOutline.apply(o) : 0).sum();
```

Comment:
surely you don't need that o != null check because of the filter above?

also, please use whitespace around operators

Reply:
changed to  .mapToInt(o -> getItemsFromOutline.apply(o)).sum();